frontend/src/App.css

```diff
@@ -28,6 +28,36 @@
   color: #61dafb;
 }
 
```

Comment:
Ezeknek a formázásoknak a többségére van jó Bootstrap helper class, d-flex, flex-row, justify-content-around, p-*, bg-*, text-center stb., inkább használjuk ezeket, ahol lehet, így a HTML-lel együtt tud mozogni a formázás.

Reply:
Köszönöm. bootrap felxboxot használ már. Illetve a jövőben is bootsrapet használok ahol csak lehet.